server/src/core/sms/types.ts

```diff
@@ -0,0 +1,22 @@
+export type SendStatus = 'Success' | 'Failed';
+
+export interface SendResult {
+  SMSMessageData: SMSMessageDataObj;
+}
+
+export interface SMSMessageDataObj {
+  Message: string;
+  Recipients: SMSRecipient[];
+}
+
+export interface SMSRecipient {
+  statusCode: number;
+  number: string;
+  cost: string;
+  status: SendStatus;
+  messageId: string;
+}
```

Comment:
The types look very specific to AT. We should normalize them for SocialRelief. E.g. SocialRelief doesn't need the `SMSMessageData`, `cost`, `statusCode`, etc.